test/test.js

```diff
@@ -9,11 +9,21 @@ var SAMPLE_JSON = {
   },
 };
 
-var SAMPLE_JSON_TWO = {
-  'JMON_TWO': {
-    'name': 'JMON_TWO',
-    'author': 'Bibek_TWO',
-    'url': 'https://github.com/bibekluitel/JMON_TWO',
+// var SAMPLE_JSON_TWO = {
+//   'JMON_TWO': {
+//     'name': 'JMON_TWO',
+//     'author': 'Bibek_TWO',
+//     'url': 'https://github.com/bibekluitel/JMON_TWO',
+//   },
+// };
+
+var SAMPLE_JSON_3 = {
+  'JMON': {
+    'name': {
+      'people': 'bibek',
+    },
+    'author': 'Bibek',
+    'url': 'https://github.com/bibekluitel/JMON',
   },
 };
 
```

Comment:
At line 30, looks like an old commit, why we have added this lint ignore on this line?
I think I missed it in my previous review.

Reply:
The linter was throwing a error for long text ...for long string we can discard eslint

Reply:
No. we can make it into multi line using `+` operator.
We can ignore this for now. I will fix it in next PR.